Podfile.lock

```diff
@@ -171,6 +167,6 @@ SPEC CHECKSUMS:
   UnionFind: c33be5adb12983981d6e827ea94fc7f9e370f52d
   YapDatabase: cd911121580ff16675f65ad742a9eb0ab4d9e266
 
-PODFILE CHECKSUM: 2f847bb25e70d1d376f38cf21ae08624fa6ed67d
+PODFILE CHECKSUM: 4a6db311736b5ab36b660af813dcdf0af62da48c
 
-COCOAPODS: 1.2.1
```

Comment:
Can you please also update the Gemfile.lock when you update cocoapods? That way you and me and travis have half a chance of being on the same build tools.

Reply:
I've downgraded cocoapods locally.